.env.example

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+#
+# Exile Server Manager environment variables
+# These are required in order for ESM to function
+#
+
+# Your Discord bot token (https://discord.com/developers/applications)
+export DISCORD_TOKEN=""
+
+# Your Steam API token (https://steamcommunity.com/dev)
+export STEAM_TOKEN=""
+
+# A JSON array of Discord IDs (as strings)
+# For example: "['123457342341','1234515342541']"
+# This allows the Discord users in the list to use development commands/functionalities
+export DEV_ALLOWLIST="[]"
+
+# The environment - Leave as development
+export ESM_ENV="development"
+
+###############################################################################
+# Only needed for automated tests until @esm v2 is released
```

Comment:
are we officially in V2 and this PR is ongoing support, or is this PR a step toward the full V2 release?

Reply:
"V1" and "V2" refer to the version of the A3 server mod (@esm). The v2 code can be found here: https://github.com/itsthedevman/esm_arma